CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Version 2.12.1 - 2020-12-19
+
+### Fixed
+- Fixed a `System.NullReferenceException` when setting _rawOuterXml field in `DocumentFormat.OpenXml.OpenXml` (#818)
+- Added regression tests in `DocumentFormat.OpenXml.Tests.OpenXmlElementTest2` to check for the mentioned exceptions in the future (#818)
```

Comment:
This line is unneeded. I try to keep the changelog focused on observable changes to the library